src/wrappers/express.js

```diff
@@ -24,9 +24,11 @@ const { methods } = require('../consts');
 function expressMiddleware(req, res, next) {
     // Check if endpoint is ignored
     utils.debugLog('Epsagon Express - starting express middleware');
+    const originalAsyncId = asyncHooks.executionAsyncId();
 
     if (shouldIgnore(req.originalUrl, req.headers)) {
         utils.debugLog(`Ignoring request: ${req.originalUrl}`);
+        traceContext.destroyAsync(originalAsyncId, true);
```

Comment:
I think you should call set reference and then pass the current async I'd (otherwise some stuff won't be caught)